internal/integration/unified/event_verification.go

```diff
@@ -488,7 +497,27 @@ func getNextTopologyDescriptionChangedEvent(
 		return nil, nil, errors.New("no topology description changed event published")
 	}
 
-	return events[0], events[:1], nil
+	return events[0], events[1:], nil
```

Comment:
Were these tests not running correctly? Do you know why this wasn't causing an error?

Reply:
We didn't have multiple events previously.